### PR TITLE
SQL: fix double comparison in tests

### PR DESCRIPTION
**Pull request by unrealwork**
# SQL: fix double comparison in tests 
- [x] fix tests

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/Util.java [hunk 1]
```diff
@@ -96,6 +94,14 @@ public static Long getMillis(String date) throws ParseException {
         return parseDate(date).getTime();
     }
 
+    public static <T> List<List<T>> twoDArrayToList(T[][] twoDArray) {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/Util.java]: Can we avoid the unknown D shortcut?

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlLimitParamTest.java [hunk 2]
```diff
@@ -62,9 +66,11 @@ public void testRequestGraterLimit() {
 
         StringTable resultTable = response.readEntity(StringTable.class);
 
-        List<String> expectedColumn = Arrays.asList("0.0", "1.0", "2.0");
+        String[][] expectedRows = {
+                {"0.0"}, {"1.0"}, {"2.0"}
+        };
 
-        assertTableContainsColumnValues(expectedColumn, resultTable, "value");
+        assertTableRows(expectedRows, resultTable);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlLimitParamTest.java]: please use more verbose method name (like `assertTableRowsExist`)

**Comment by Sild** [comment]
lgtm

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**